README.md

```diff
@@ -23,3 +27,20 @@ To use the Python module:
 ``` sh
 $ export PYTHONPATH=build/lib/python:$PYTHONPATH
 ```
+
+### Using Conda
+
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/vampyr/badges/version.svg)](https://anaconda.org/conda-forge/vampyr)
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/vampyr/badges/latest_release_date.svg)](https://anaconda.org/conda-forge/vampyr)
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/vampyr/badges/downloads.svg)](https://anaconda.org/conda-forge/vampyr)
+
+To install VAMPyR in a Conda environment `myenv`:
+
+    $ conda create -n myenv
+    $ conda activate myenv
+    $ conda install -c conda-forge vampyr               # latest version (OpenMP)
+    $ conda install -c conda-forge vampyr=0.1.0rc0      # tagged version (OpenMP)
```

Comment:
I would add: "Note that the conda-forge package is _always_ built with OpenMP support enabled in the MRCPP backend."